jenkins/python-setup.sh

```diff
@@ -1,23 +1,22 @@
-_prefix="${1}"
+#!/bin/bash
 
-if [[ -z "${_prefix}" ]]; then
-    printf -- "Missing prefix argument!" >&2
-    exit 1
-fi
+# This script is run by jenkins/run-unittests to set up the Python environment
```

Comment:
Perhaps adjust the comment to reference this is used ahead of both `run-*` scripts?